nucleus/deploy/client.py

```diff
@@ -139,11 +144,15 @@ def create_model_bundle(
                 "tensorflow_version": Version of tensorflow, e.g. "2.3.0". Only applicable if framework_type is tensorflow
         """
 
-        if (model is not None and load_model_fn is not None) or (
-            model is None and load_model_fn is None
-        ):
+        check_args = [
+            predict_fn_or_cls is not None,
+            model is not None,
+            load_model_fn is not None,
+        ]
+
+        if sum(check_args) != 1:
             raise ValueError(
-                "Exactly one of model and load_model_fn should be non-None"
+                "Exactly one of `model` or `load_model_fn` or `predict_fn_or_cls` should be non-None"
```

Comment:
Should we check that `load_predict_fn` doesn't get passed if a user passes `predict_fn_or_cls`?